templates/master.erb

```diff
@@ -1,29 +1,13 @@
-# ZooKeeper URL (used for leader election amongst masters)
-ZK="<%= @zookeeper %>"
+MESOS_work_dir="<%= @work_dir %>"
+MESOS_zk="<%= @zookeeper %>"
+MESOS_quorum=<%= @quorum %>
+MESOS_ip="<%= @listen_address %>"
+MESOS_port=<%= @master_port %>
+MESOS_cluster="<%= @cluster %>"
+MESOS_log_dir="<%= @log_dir %>"
+MESOS_logbufsecs=1
+MESOS_logging_level="INFO"
```

Comment:
We should decide whether we want to configure Mesos via properties (`/etc/mesos-master/*` and `/etc/mesos-slave/*`) or by using ENV `MESOS_{variables}`. I think supporting both at the same time is very confusing. CLI arguments are out of questions, we expect people run Mesos as service using systemd, sysvinit or whatever else. When someone checks Mesos machine configuration will he look into `/etc/default/mesos-slave` or `/etc/mesos-slave`? Each Mesos version adds new parameters, it's problematic to add new variables each time this happens.